src/test/java/org/betonquest/betonquest/modules/logger/custom/chat/ChatFormatterTest.java

```diff
@@ -0,0 +1,126 @@
+package org.betonquest.betonquest.modules.logger.custom.chat;
+
+import org.betonquest.betonquest.api.config.QuestPackage;
+import org.betonquest.betonquest.modules.logger.BetonQuestLogRecord;
+import org.betonquest.betonquest.modules.logger.util.BetonQuestLoggerService;
+import org.bukkit.plugin.Plugin;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import java.util.logging.Level;
+import java.util.logging.LogRecord;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.*;
+
+/**
+ * A test for the {@link ChatFormatter}.
+ */
+@ExtendWith(BetonQuestLoggerService.class)
+class ChatFormatterTest {
```

Comment:
It might be possible to improve readability of these tests with [parametrized tests](https://junit.org/junit5/docs/current/user-guide/#writing-tests-parameterized-tests).